pkg/volume/util/util_test.go

```diff
@@ -263,3 +265,42 @@ func TestZonesToSet(t *testing.T) {
 		}
 	}
 }
+
+func TestUnmountMountPath(t *testing.T) {
+
+	tmpDir1, err1 := utiltesting.MkTmpdir("umount_test1")
+	if err1 != nil {
+		t.Fatalf("error creating temp dir: %v", err1)
+	}
+	defer os.RemoveAll(tmpDir1)
+
+	tmpDir2, err2 := utiltesting.MkTmpdir("umount_test2")
+	if err2 != nil {
+		t.Fatalf("error creating temp dir: %v", err2)
+	}
+	defer os.RemoveAll(tmpDir2)
+
+	// Second part: want no error
+	tests := []struct {
+		mountPath    string
+		corruptedMnt bool
+	}{
+		{
+			mountPath:    tmpDir1,
+			corruptedMnt: true,
+		},
+		{
+			mountPath:    tmpDir2,
+			corruptedMnt: false,
+		},
+	}
+
+	fake := &mount.FakeMounter{}
+
+	for _, tt := range tests {
+		err := unmountMountPath(tt.mountPath, fake, false, tt.corruptedMnt)
```

Comment:
What this call actually tests? What is expected output / behavior of unmountMountPath and how is it tested here?

Reply:
the tests verify that either corruptedMnt is ture or false, unmountMountPath will return nil (means success).